includes/class-coblocks-block-assets.php

```diff
@@ -187,22 +187,14 @@ public function frontend_scripts() {
 		}
 
 		// Lightbox.
-		if ( has_block( $this->slug . '/gallery-masonry' ) || has_block( $this->slug . '/gallery-stacked' ) ) {
+		if ( has_block( $this->slug . '/gallery-masonry' ) || has_block( $this->slug . '/gallery-stacked' ) || has_block( $this->slug . '/gallery-collage' ) ) {
 			wp_enqueue_script(
 				$this->slug . '-lightbox',
 				$dir . $this->slug . '-lightbox' . COBLOCKS_ASSET_SUFFIX . '.js',
 				array( 'jquery' ),
 				COBLOCKS_VERSION,
 				true
 			);
-
-			wp_enqueue_script(
-				$this->slug . '-masonry',
-				$dir . $this->slug . '-masonry' . COBLOCKS_ASSET_SUFFIX . '.js',
-				array( 'jquery', 'masonry', 'imagesloaded' ),
-				COBLOCKS_VERSION,
-				true
-			);
```

Comment:
This seems to be a bug. Removed this snippet because it caused the masonry script to load even if masonry block is not in use. On line `150` the script is enqueued appropriately.